inc/settings.php

```diff
@@ -414,6 +491,51 @@ function render_banner_message() {
 	] );
 }
 
+/**
+ * Get the filterable array of policy page values that can be created.
+ *
+ * @return array An array of policy page types.
+ */
+function get_allowed_policy_page_values() : array {
+	/**
+	 * Filter the array of allowed policy pages we can create.
+	 *
+	 * This is used by the render_secondary_button function. The default pages that can be created are privacy_policy and cookie_policy.
+	 *
+	 * @var array An array of allowed policy page values.
+	 */
+	return apply_filters( 'altis.consent.allowed_policy_page_values', [ 'cookie_policy', 'privacy_policy' ] );
+}
+
+/**
+ * Display a secondary button.
+ *
+ * Used to create the Create Policy Page buttons, but can be filtered and used for other things.
+ *
+ * @param string $button_text The text to display in the button.
+ * @param string $value       The button value. On the settings page, this is used to determine the type of policy page the buttons create.
+ * @param string $type        The html button type. The default value is 'submit', and valid values are 'submit', 'reset', and 'button'. Invalid values revert to 'submit'.
+ */
+function render_secondary_button( string $button_text, string $value = 'privacy_policy', string $type = 'submit' ) {
+	// Make sure the button type is valid. Invalid types are reset to 'submit'.
+	$type = in_array( $type, [ 'submit', 'reset', 'button' ], true ) ? $type : 'submit';
+
+	/**
+	 * The html name of the button we're creating.
+	 *
+	 * The default here is 'create_policy_page' because that's what we're using it for here. However, this function could be used to create other types of buttons with unique names by using this filter.
+	 *
+	 * @var string The button element name value.
+	 */
+	$name = apply_filters( 'altis.consent.admin_secondary_button_name', 'create_policy_page' );
```

Comment:
What other use cases are you thinking of here? I'd be more inclined to just remove it if there are no immediate obvious reasons for it or make it a parameter of the function instead.

Reply:
I guess I was thinking that the `render_secondary_button` function, given the generic-ness of the name, could be used for _any_ type of button that triggers _anything_, but that's dependent upon the parameters that are passed into the actual button code. To make that as extensible as possible, the last piece would be to specify the name, but I suppose, rather than a filter here, it could just be another function parameter.

Reply:
I think it's unlikely that anything else would make use of such a generic function provided by this plugin. Plus if anyone used that filter it would change all instances of buttons that use this function. 

There is a template tag in core WP called `submit_button()` that you could perhaps use instead.

Reply:
I forgot that function existed 🙃 🤦

Reply:
Ah right, `input` vs `button` 👍  - you'd have to use a different name value to achieve the same effect rather than a different `value` attribute. As you were 😄